.travis.yml

```diff
@@ -1,14 +1,30 @@
 language: csharp
 
+sudo: required
+dist: trusty
+
+env:
+  - CLI_VERSION=1.0.0-preview1-002702
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+
```

Comment:
That was all I needed to get as far as the dotnet tool running, and successfully restoring and building packages. It's only the test that fails.